### PR TITLE
recipe for rootpath

### PR DESCRIPTION
**Pull request by mediocretech**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mediocretech** [comment]
@conda-forge/help-python, ready for review!

File: recipes/rootpath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rootpath-{{ version }}.tar.gz
+  sha256: ecc3f9de280ff11c6ade0e956229e4dcb74c1ead5efde8579ba721e6459c518f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/rootpath/meta.yaml:15]: I believe this one can be noarch. What is the reason to skip Windows?

Comment (mediocretech) [reply recipes/rootpath/meta.yaml:15]: @ocefpaf thanks for reviewing!

It's due to a missing nested dependancy `[nothing provides blessings needed by colour-runner-0.1.1-py_0](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=838854&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4)`.  Confirming that there is no [Windows build for Blessings](https://anaconda.org/conda-forge/blessings).

Comment (ocefpaf) [reply recipes/rootpath/meta.yaml:15]: Yep, in that case you should make it noarch. Failing on Windows due to a missing dependency is fine. Noarch not only saves CI time but also, then that dependency is available on Windows, nothing is needed in this recipe to make it available there.

Comment (mediocretech) [reply recipes/rootpath/meta.yaml]: Thanks @ocefpaf, I'm catching the pattern now. noarch coming up.

**Pull request merged**